packages/features/lend/src/lend-card-view.tsx

```diff
@@ -4,40 +4,37 @@ import { CardContainer } from '@notional-finance/shared-web';
 import { CurrencyFixed } from '@notional-finance/mui';
 import { ThemeProvider } from '@mui/material';
 import { useNotionalTheme } from '@notional-finance/styles';
-import { defineMessages, FormattedMessage } from 'react-intl';
+import { defineMessage, FormattedMessage } from 'react-intl';
 import { useUserSettingsState } from '@notional-finance/user-settings-manager';
 
 export function LendCardView() {
   const { themeVariant } = useUserSettingsState();
   const themeLanding = useNotionalTheme(themeVariant, 'landing');
   const { cardData } = useAllMarkets();
 
-  const heading = defineMessages({
-    fixed: {
-      defaultMessage: 'Lend Crypto at Fixed Rates',
-      description: 'page heading',
-    },
-    variable: {
-      defaultMessage: 'Fix Your Variable Rate',
-      description: 'page heading',
-    },
+  const heading = defineMessage({
+    defaultMessage: 'Fixed Rate Lending',
+    description: 'page heading',
   });
-  const subtitle = defineMessages({
-    fixed: {
-      defaultMessage:
-        'Build a stable portfolio with fixed rate income on your assets. Lock in your yield for up to one year or exit early without penalty at the market rate.',
-      description: 'page subtitle',
-    },
-    variable: {
-      defaultMessage:
-        'Use your cTokens directly on Notional to swap your variable rate loan to a fixed rate and lock in your returns.',
-      description: 'page subtitle',
-    },
+
+  const subtitle = defineMessage({
+    defaultMessage: `Fix your rate and earn guaranteed returns with peace of mind.`,
+    description: 'page subtitle',
+  });
+
+  const docsText = defineMessage({
+    defaultMessage: 'Read fixed lend docs.',
+    description: 'docs link',
   });
 
   return (
     <ThemeProvider theme={themeLanding}>
-      <CardContainer heading={heading.fixed} subtitle={subtitle.fixed}>
+      <CardContainer
+        heading={heading}
+        subtitle={subtitle}
+        linkText={docsText}
+        docsLink="https://docs.notional.finance/notional-v2/notional-v2-basics/lending"
```

Comment:
@T-Woodward is this the right link?

Reply:
https://docs.notional.finance/notional-v2/what-you-can-do/fixed-rate-lending